### DISCUSSION TITLE
Not able to uninstall Appium 2.0.0-beta.46 on mac Monterey

### DISCUSSION THREAD

**Discussion by vnuta9** [appium/appium#17677] [Q&A] [answered]
I used npm to install appium 2.0.0-beta.46.
After that I installed appium 1.22, but the appium -v still shows 2.0.0-beta.46.
So I tried various options

- npm uninstall -g appium@2.0.0-beta.46
- sudo npm uninstall -g appium@2.0.0-beta.46
- sudo npm uninstall -g appium@beta
- sudo npm uninstall -g appium@next

Plus 

- npm cache clean --force

But still, I am getting
$ **appium -v
2.0.0-beta.46**

When I did 
**$which appium,** I get
**/usr/local/bin/appium**

I rebooted the system twice, but no use.
What is the best way to do the uninstall the 2.0.0-beta.46 and go back to 1.22 ?

Appreciate the help

**Comment by jlipps** [comment]
Sounds like you have Appium installed some other way than via NPM if it's still there. You should figure out what put things into `/usr/local/bin/appium` and follow uninstall procedures for that. Or just remove the file/symlink.

**Comment by vnuta9** [comment, marked as answer]
Thanks for the reply.
I don't remember using any other means to install appium. I took risk and installed a trialware 'PowerMyMac' tool and with it's help, I cleaned up appium (deskdop version fragments). The desktop version is obviously not 2.0.0.beta but 1.22 version. Ultimately I don't know what happened, appium is cleaned up. I did the similar thing for node also and cleaned it also. 
I am not really confident in doing the above, but being desperate I did it. Plus it made me feel that many tools we install on mac are not having clean uninstaller programs. There are different tools like Homebrew, sdkman, npm, nvm, curl etc., they all do the same job, but there are many idiosyncrasies, which are a night mare most of the times.
Any how that is how I moved, and right now after installing node using nvm (instead of home brew), I am going to install appium, the latest non beta version.
